packages/sentry/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
   "extends": "../../tsconfig.json",
-  "files": ["./src/index.ts"]
+  "files": ["./src/index.ts"],
+  "include": ["./src/globals.d.ts"]
```

Comment:
This is important in case we have custom declaration files

<img width="687" alt="image" src="https://user-images.githubusercontent.com/1110551/59109212-f1d7b700-893c-11e9-97de-a7a2eb4c73a5.png">